docs/mesos/UsageGuideUnifiedCNI.md

```diff
@@ -30,44 +30,102 @@ EOF
 ```
 > Note: If you have used the Docker-Compose demo to launch a cluster, this network configuration has already been created for you.
 
-Mesos will actively scan that directory when launching containers, so without needing to restart the Agent process, you are now ready to launch Calico-CNI Mesos tasks.
+Mesos-Agent loads these network configurations into memory at launch, so you will need to restart the Agent process for your new configuration to take effect.
 
 ## 2. Launch a Calico-Networked Task
-#### Using Mesos-Execute
-With our network configured, we can use `mesos-execute` to launch a task on the CNI network you just configured by setting `--networks` to the name of the network you just configured:
+#### a.) Using Mesos-Execute
+To quickly test that Calico's Mesos integration is working, use `mesos-execute` to launch a task on the created CNI network by setting `--networks`:
```

Comment:
I quite liked having the original preamble that you used to have here - but I think that should go directly below the 2. heading .... i.e. a bit to say you've just created a network, now test it using one or more of the following methods.